bcgov/biohubbc
# TechDebt: Map container refactor

## Links to Jira Tickets

[SIMSBIOHUB-351](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-351)

## Description of Changes

- Updated `DrawControls.tsx`
- Removed old `MapComponent.tsx`
- Removed old `MapBoundary.tsx`
- Removed project location code from App/ Api
- Updated OpenApi schema to not require locations on project updated/ create
- Updated Survey Study area details to not require a map component
- Removed old search page

## Testing Notes

- You should be able to create and edit projects as normal
- You should be able to create a survey with locations (map should still be working, draw controls should be functional)
- Add a marked animal with a capture event (map should still be working)
- Add a telemetry device to that animal, open the database and change the `critterbase_critter_id` to `826aa7b4-31cf-44d5-80cd-fc159e42d7e7`, then view telemetry 
- Open Manage Observations and add a sampling site(map should still be working)
- Add a few observations and navigate back to survey details (map should still be working)
